gui/GameConqueror.py

```diff
@@ -23,12 +23,11 @@
 
 import sys
 import os
-import re
+import argparse
 import struct
 import tempfile
 import platform
 import threading
-import time
```

Comment:
Dropping `re` and `time` imports is not documented. Are you 100% sure that none of this functionality is included?